pkg/util/kube/groupresource.go

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
This PR isn't using this file, which is ok for now. I think I'd like to move it to a standalone package that doesn't import anything other than `schema`.